homeassistant/components/zha/sensor.py

```diff
@@ -206,43 +93,34 @@ def battery_percentage_remaining_formatter(value):
 
 async def make_sensor(discovery_info):
     """Create ZHA sensors factory."""
-    return Sensor(**discovery_info)
+
+    zha_dev = discovery_info["zha_device"]
+    channels = discovery_info["channels"]
+
+    entity = ZHA_ENTITIES.get_entity(DOMAIN, zha_dev, channels, Sensor)
+    return entity(**discovery_info)
 
 
 class Sensor(ZhaEntity):
     """Base ZHA sensor."""
 
+    _decimals = 1
+    _device_class = None
+    _divisor = 1
     _domain = DOMAIN
```

Comment:
Should we make the domain attribute non private to not have to disable that pylint rule?

Reply:
I think we should remove that attribute. It's not needed anymore I think since #28362
I'm thinking if we should keep the `MatchRule` as an attribute of the class instead.

Reply:
i'll remove `_domain` attr in a separate PR.

Reply:
Don't access it in https://github.com/home-assistant/home-assistant/pull/30108/commits/c4a8e9b6e12519404781537f68226f0a1fe7cf2d